components/script/script_thread.rs

```diff
@@ -570,6 +570,9 @@ pub struct ScriptThread {
 
     /// The Webrender Document ID associated with this thread.
     webrender_document: DocumentId,
+
+    /// FIXME(victor):
```

Comment:
@ceyusa could you elaborate more on what needs to be fixed here exactly, please?

Reply:
I used the wrong tag. It is not a fixme but a xxx, perhaps. What I mean is it would be nice to rethink this design. Perhaps it is better to create a proxy object to keep the RenderApiSender for the servo-media, rather than pass it all along from constellation to document to html element.

Reply:
I think process isolation wise script should not have access to this, cc @asajeffrey 

A proxy approach would be nice. Ideally the network task would directly talk to WR, maybe via constellation